Header header

string parentNode

string errorKey

string errorDescription

uint8 errorType
uint8 OUT_OF_RANGE = 1
uint8 INVALID_VALUE = 2

uint8 errorLevel
uint8 OK = 1
uint8 WARNING = 2
uint8 ERROR = 3